mat64/io_test.go

```diff
@@ -4,7 +4,20 @@
 
 package mat64
 
-import "testing"
+import (
+	"encoding"
+	"math"
+	"testing"
+
+	"github.com/gonum/blas/blas64"
+)
+
+var (
+	_ encoding.BinaryMarshaler   = (*Dense)(nil)
+	_ encoding.BinaryUnmarshaler = (*Dense)(nil)
+	_ encoding.BinaryMarshaler   = (*Vector)(nil)
+	_ encoding.BinaryUnmarshaler = (*Vector)(nil)
+)
 
 func TestDenseRW(t *testing.T) {
```

Comment:
Rename to something more predictable? `TestDenseIO` or `TestDenseMarshalUnmarshall`?